### ISSUE TITLE
[Docs] Function length sniff configuration

### ISSUE THREAD

**Issue by Blitheness** [nunomaduro/phpinsights#483]
The [documentation ](https://phpinsights.com/insights/architecture.html#function-length) for `\SlevomatCodingStandard\Sniffs\Files\FunctionLengthSniff::class` is using an incorrect key in the example configuration.

Instead of "maxLength", it should be "maxLinesLength" as per [FunctionLengthSniff.php:44](https://github.com/slevomat/coding-standard/blob/0012151f0f0fc33709472667368c568dffda2839/SlevomatCodingStandard/Sniffs/Functions/FunctionLengthSniff.php#L44)

Raising it as an issue instead of a PR because I couldn't find a repo for the docs website.

**Comment by 50bhan** [comment]
@Blitheness Yes, you are right! I've just changed this property name in the `Slevomat` package to be more consistent but forgot to change it here as well 😅  Thank you for this 👍
